cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialTest.scala

```diff
@@ -54,7 +54,7 @@ class SpatialTest extends DocumentingTest {
           |* Each point will also be associated with a specific <<cypher-spatial-crs,Coordinate Reference System>> (CRS) that determines the meaning of the values in the _Coordinate_.
           |* Instances of _Point_ and lists of _Point_ can be assigned to node and relationship properties.
           |* Nodes with _Point_ or _List(Point)_ properties can be indexed using a spatial index. This is true for all CRS (and for both 2D and 3D).
-          |  There is no special syntax for creating spatial indexes, as it is supported using the existing <<administration-indexes-create-a-single-property-index-for-nodes,indexes>>.
+          |  There is no special syntax for creating spatial indexes, as it is supported using the existing <<administration-indexes-create-a-single-property-b-tree-index-for-nodes,indexes>>.
```

Comment:
This is not true with introduction of point indexes. It is still true for btree, but I guess the point index should be mentioned here?

Reply:
I know I discussed this line with @Lojjs at some point but can't find that discussion now.

I think we decided that it was part of the planner teams documentation of index usage more than our part of create/drop commands for them

Reply:
Yeah, I did comment on the same thing, but not sure if it was on this PR or another one... Might be worth to reach out to the planner team so they add a note to their point Trello card about this, it will be easily missed otherwise I think